lib/rails_state_machine/model.rb

```diff
@@ -0,0 +1,21 @@
+module RailsStateMachine
+  module Model
+    def self.included(base)
+      base.extend ClassMethods
+    end
+
+    module ClassMethods
+      def state_machine(&block)
+        StateMachine.new(self).configure(&block)
+      end
```

Comment:
After `configure` is called, and itself called `register_state_machine`, this class method will be overwritten and the state machine can be read from the class using the `state_machine` class method, right?

Maybe that's a bit of a weird/unexpected pattern, but I'm fine with it.
To avoid confusion, the class method *could* maybe configure only when a block was given, and return the state machine if none was given -- but defining twice for a model won't work anyway, so we probably should not bother. There might be people with strong opinions about that, though. ;)